src/main/java/com/coravy/hudson/plugins/github/GithubLinkAnnotator.java

```diff
@@ -43,8 +45,8 @@ void annotate(final GithubUrl url, final MarkupText text, final Entry change) {
         if (change instanceof GitChangeSet) {
             GitChangeSet cs = (GitChangeSet) change;
             final String id = cs.getId();
-            text.wrapBy("", " (<a href='" + url.commitId(id)
-                        + "'>commit: " + id.substring(0, Math.min(id.length(), 7)) + "</a>)");
+            final String innerText = "commit: " + id.substring(0, Math.min(id.length(), 7));
+            text.wrapBy("", format(" (<a href='%s'>%s</a>)", url.commitId(id), innerText));
```

Comment:
Unfortunately, I'm not an IDEA user.  Can you provide your preferred change to that line?